src/styles/style-guide.scss

```diff
@@ -6,6 +6,21 @@
 @import "style";
 @import "components/Swatch";
 
+.container {
```

Comment:
This is all in the other PR https://github.com/humanmade/hm-pattern-library/pull/73 - perhaps we can close